rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/PostgresqlTableMetaCache.java

```diff
@@ -102,7 +102,7 @@ private TableMeta resultSetMetaToSchema(DatabaseMetaData dbmd, String tableName)
          * select * from "Sel'ect"
          */
         if (null != schemaName) {
-            schemaName = schemaName.replaceAll("(^\")|(\"$)", "");
+            schemaName = schemaName.replaceAll("(^\")|(\"$)", "").toLowerCase();
```

Comment:
why if schema name not null, lower case the schema name?
How about the situation like below?
```
CREATE DATABASE "SeAtA"
```

Reply:
fix:
select * from TEST.test
select * from test.TEST
select * from "Test".test
select * from "Test"."Select"